.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,6 @@
+name: "Custom CodeQL Config"
+
+paths-ignore:
+  - core/tests/data
+  - core/controllers/acl_decorators.py
+  - scripts/release_scripts/update_configs.py
```

Comment:
Can you add a comment explaining why we ignore these, and if we are only ignoring them temporarily, a TODO with issue number for removing them?

Reply:
Done

Reply:
The _core/controllers/acl_decorators.py_ is already fixed I think, can you try to remove it from here?

Reply:
Done